### ISSUE TITLE
Bivariate Toy Example does not work with Julia 0.5.2 (Jupyter notebook)

### ISSUE THREAD

**Issue by mgiugliano** [joshday/AverageShiftedHistograms.jl#13]
While the Univariate Toy Example works without problems, the Bivariate Toy Example - as mentioned in the documentation (see below) - does not work with Julia 0.5.2 (Jupyter notebook):

```
using AverageShiftedHistograms
using Plots; pyplot()

x = randn(10_000)
y = x + randn(10_000)

o = ash(x, y)

plot(o)
```

It returns the error:

In convertToAnyVector, could not handle the argument types: (AverageShiftedHistograms.BivariateASH,)

 _in macro expansion at /Users/M/.julia/v0.5/Plots/src/series.jl:129 [inlined]
 in apply_recipe(::Dict{Symbol,Any}, ::Type{Plots.SliceIt}, ::Void, ::AverageShiftedHistograms.BivariateASH, ::Void) at /Users/M/.julia/v0.5/RecipesBase/src/RecipesBase.jl:238
 in _process_userrecipes(::Plots.Plot{Plots.PyPlotBackend}, ::Dict{Symbol,Any}, ::Tuple{AverageShiftedHistograms.BivariateASH}) at /Users/M/.julia/v0.5/Plots/src/pipeline.jl:73
 in _plot!(::Plots.Plot{Plots.PyPlotBackend}, ::Dict{Symbol,Any}, ::Tuple{AverageShiftedHistograms.BivariateASH}) at /Users/M/.julia/v0.5/Plots/src/plot.jl:171
 in plot(::AverageShiftedHistograms.BivariateASH) at /Users/M/.julia/v0.5/Plots/src/plot.jl:46_

**Comment by joshday** [comment]
It doesn't error on Julia 0.6, but it's not right either:

<img width="596" alt="screen shot 2017-06-22 at 1 08 23 pm" src="https://user-images.githubusercontent.com/8075494/27453604-eb7f0b4e-574b-11e7-9880-92dfc8f08678.png">

I'll take a look.

**Comment by joshday** [comment]
This (my result) appears to be a problem with heatmap.  I changed the seriestype to contour.  

<img width="595" alt="screen shot 2017-06-23 at 3 03 16 pm" src="https://user-images.githubusercontent.com/8075494/27501897-1d712432-5825-11e7-9b3a-728ab77ea8b8.png">

As far as your original issue, I remembered I did not have a plot recipe for bivariate ASH until after I stopped supporting 0.5.  If `o` is your object, `contour(o.rngx, o.rngy, o.z)` should be a workaround.
